src/api_midlevel.jl

```diff
@@ -185,3 +185,33 @@ function write_chunk(dataset_id, index::Integer, buf::Union{DenseArray,Base.Fast
     offset = [reverse(get_chunk_offset(dataset_id, index))...]
     write_chunk(dataset_id, offset, buf; dxpl_id = dxpl_id, filter_mask = filter_mask)
 end
+
+"""
+    is_filter_available(filter_or_id)
+
+Given a subtype of `Filters.Filter` or the filter ID number as an integer,
+return `true` if the filter is available and `false` otherwise.
+"""
+is_filter_available(filter_or_id) = API.h5z_filter_avail(filter_or_id)
+
+"""
+    can_filter_encode(filter_or_id)
+
+Given a subtype of `Filters.Filter` or the filter ID number as an integer,
+return `true` if the filter can encode or compress data.
+"""
+function can_filter_encode(filter_or_id)

```

Comment:
`can_filter_encode`/`can_filter_decode`  read  a bit strange

what about  ?
`filter_encoder_enabled`
`filter_decoder_enabled`

Ideally, with dispatch on the `Filter` type, we would use more generic names   but the fact they also take an `Integer` makes that less feasible.

Reply:
thoughts on this?

Reply:
Or maybe `is_encoder_enabled`?

Reply:
that works too, although 
![image](https://user-images.githubusercontent.com/4319522/154106764-6244422c-db43-45b4-a3b1-3a93bf406c17.png)

I'm not sure if consistency with base naming matters that much, but just  thought to mention.